docs/api/README.md

```diff
@@ -83,13 +111,31 @@ If you do not care about exact dimensions, rather size class, it is better to us
 
 ```javascript
 // To make use of width and height in our component, we just access width and 
-// height as props:
-const Info = ({ width = 0, height = 0}) => (
-  <Text>{width}pt x {height}pt</Text>
+// height by using WithContainerDimensions helper:
+const Info = () => (
+  <WithContainerDimensions>{(width, height) => (<Text>{width}pt x {height}pt</Text>)}</WithContainerDimensions>
```

Comment:
This should be broken into multiple lines to be consistent with the rest of documentation.